doctrine/custom_dql_functions.rst

```diff
@@ -5,7 +5,7 @@ How to Register custom DQL Functions
 ====================================
 
 Doctrine allows you to specify custom DQL functions. For more information
-on this topic, read Doctrine's cookbook article "`DQL User Defined Functions`_".
+on this topic, read Doctrine's cookbook article `DQL User Defined Functions`_.
```

Comment:
This should normale work with the `"`, but I agree, that its not really necessary here.

WDYT @javiereguiluz ?

Reply:
Yes, let's remove the unneeded quotes here.